src/components/application_manager/src/commands/mobile/delete_file_request.cc

```diff
@@ -87,24 +102,40 @@ void DeleteFileRequest::Run() {
   full_file_path += sync_file_name;
 
   if (file_system::FileExists(full_file_path)) {
+    LOG4CXX_DEBUG(logger_, "File exists! Path is " << full_file_path);
```

Comment:
@AByzhynar do we rely need to emphasize that file exists?

Reply:
@LuxoftAKutsan All these logs were added just for debugging purposes. Now they are deleted as redundant. 
Please check 5202f40d6d9bc3b62c6b6044e99fef849842edf3